darwin/dataset/upload_manager.py

```diff
@@ -162,9 +162,19 @@ def serialize(self):
 
     def serialize_v2(self):
         return {
-            "slots": [{"file_name": self.data["filename"], "slot_name": "0", "fps": self.data["fps"]}],
+            "slots": [
+                {
+                    "file_name": self.data["filename"],
+                    "slot_name": "0",
+                    "fps": self.data["fps"],
+                    "extract_views": self.data["extract_views"],
```

Comment:
This change makes `extract_views` mandatory for SDK users, which it shouldn't. Please see how I refactored this here: https://github.com/v7labs/darwin-py/pull/486 and let's make this in similar way. I'm hoping my PRs will get merged soon, only waiting for QA now. Which yours will be waiting for too actually.

Reply:
Could you update this ? change has been made